style.css

```diff
@@ -0,0 +1,151 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+header {
+  padding: 80px 34px 82px 34px;
+  border-bottom-left-radius: 80px;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  background-image: url("./header-shapes\ mobile@2x.svg");
+  background-color: #e5e5e5;
+  position: relative;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
```

Comment:
- [ ] Great job so far 👍, however, it's better to set the background image in the `section` rather than the body, so that later on when you add another section it won't mess up your style.